site/content/internal/mobile-build-process/_index.md

```diff
@@ -2,26 +2,73 @@
 title: Mobile Build Process
 heading: "The Mattermost Mobile Build Process"
 description: "Learn what there is to know about the mobile build process for Mattermost's applications."
-date: 2019-03-14T09:05:00-05:00
+date: 2023-03-09T09:05:00-05:00
 weight: 31
 ---
 
-Steps to cut the mobile build:
+# 1. Pre-requisites
 
-1. Set the proper [version number](/internal/mobile-build-process/bump-version-number/) if needed.
-{{%note "Version number" %}}Normally the version number will be set after a release, but sometimes is needed when cutting a dot release.{{%/note%}}
+In order to run all the Fastlane scripts, you will need an Apple machine. The steps can be run manually, but the scripts make things much more easy.
 
-2. Set the proper [build number](/internal/mobile-build-process/bump-build-number/) if needed.
-{{%note "Build number" %}}The build number has to be incremented each time for builds that are going to be published to the stores.{{%/note%}}
+You must have ruby 2.7. You can use [`rbenv`](https://github.com/rbenv/rbenv) to manage your ruby version.
```

Comment:
We will be bumping this to ruby 3.2 as 2.7 reaches eol in three weeks and 2 days